platforms/shared/charts/besu-connector/templates/service.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: {{ .Values.metadata.namespace }}
+  name: {{ .Values.service.name | quote }}
+  annotations:
+    app: consortiumchain
+  # version: '1'
```

Comment:
Connectors will generally stay outside the ledger environment, thus having ambassador mappings is must